### DISCUSSION TITLE
How to become verified creator of a github action?

### DISCUSSION THREAD

**Discussion by jirfag** [community/community#25265] [Actions]
<p>E.g. like <a href="https://github.com/marketplace/actions/github-action-for-cloudflare-workers" rel="nofollow noopener">this action</a>.<br>
I would like to get a badge “Verified Creator” for my action.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/jirfag">@jirfag</a>,</p>
<p>Currently, there is no way to request verification. Creators are verified because they have an existing relationship GitHub, and GitHub has worked closely with the creator to create their actions. We are continuing to evaluate this process, and we may open it up more broadly in the future.</p>
<p>Thanks</p>

**Comment by miguelbemartin** [comment]
<p>any update on this one? is there any way to get verified the github action?</p>

**Comment by kalsky** [comment]
<p>I guess this should be the answer now:</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/developers/github-marketplace/applying-for-publisher-verification-for-your-organization" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/developers/github-marketplace/applying-for-publisher-verification-for-your-organization" target="_blank" rel="noopener nofollow ugc">Applying for publisher verification for your organization - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/developers/github-marketplace/applying-for-publisher-verification-for-your-organization" target="_blank" rel="noopener nofollow ugc">//docs.github.com/en/developers/github-marketplace/applying-for-publisher-verification-for-your-organization</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by laughedelic** [comment]
<p>I also asked this through the support and the answer was that at the moment this is only available for organizations, not users.</p>

**Comment by kanadgupta** [comment]
<p>Hey <a class="mention" href="/weide-zhou">@weide-zhou</a> and co!</p>
<p>I’m currently struggling with verified actions and dug up this thread, thanks for all the info!</p>
<p>I’m working on a GitHub Action on behalf of the <code>readmeio</code> organization. The org received publisher verification (see below) over six months ago.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/c/6c106922922d47415b29b7d33f73ef3e60dc5695.png" data-download-href="/uploads/short-url/fpYJ2NWw0wYGdHb3ZpofGoHBXpj.png?dl=1" title="Screen Shot 2022-03-07 at 11.04.17 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/8854718/181089962-c1ac4c91-b968-49de-b89a-8545aaa874f9.png" alt="Screen Shot 2022-03-07 at 11.04.17 AM" data-base62-sha1="fpYJ2NWw0wYGdHb3ZpofGoHBXpj" width="690" height="320" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6c106922922d47415b29b7d33f73ef3e60dc5695_2_690x320.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6c106922922d47415b29b7d33f73ef3e60dc5695_2_1035x480.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6c106922922d47415b29b7d33f73ef3e60dc5695_2_1380x640.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6c106922922d47415b29b7d33f73ef3e60dc5695_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2022-03-07 at 11.04.17 AM</span><span class="informations">1914×888 80.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p><s>However, when viewing the Marketplace listing of the Action we recently published, it still says we’re unverified: <a href="https://github.com/marketplace/actions/rdme-sync-to-readme" class="inline-onebox" rel="noopener nofollow ugc">rdme 🦉 Sync to ReadMe · Actions · GitHub Marketplace · GitHub</a></s></p>
<p><s>Does anyone know what could be happening here? I’ve tried to get in touch with GitHub support with no luck.</s></p>
<p>Thanks so much in advance!</p>
<p>-Kanad</p>
<p><strong>EDIT</strong>: it appears that <a href="https://partner.github.com/" rel="noopener nofollow ugc">GitHub’s Partner program</a> is the only way to actually obtain verification and that publisher verification is a bit of a misnomer. More in <a href="https://github.community/t/any-risks-of-moving-already-published-action-repository/163739/7">this thread</a>!</p>

**Reply by ValentinVignal** [reply]
Hello @kanadgupta it looks like the thread https://github.com/orgs/community/discussions/ doesn't exist. Would you know what is the correct link?

**Comment by sgammon** [comment]


**Reply by josh-padnick** [reply]
@sgammon I'm so sorry, and thank you for calling this out. I must have copy/pasted into the wrong box. I'll attempt to delete the above message now.

**Reply by josh-padnick** [reply]
Deleted.

**Reply by sgammon** [reply]
@josh-padnick why would anybody be curious? Why would you post it again? It's almost like you are abusing the issues system for SEO.

**Reply by josh-padnick** [reply]
That was not my intention. I've deleted the original post in full.

**Comment by ValentinVignal** [comment]
Are there updates on this ? 
Is [GitHub’s Partner program](https://partner.github.com/) the only way to obtain verification? And is it feasible for individuals or small organization to get it?

**Comment by cheelim1** [comment]
Checking, if there's it's supported to obtain a verification for a github action published in the marketplace that is not tied to an organization?

**Reply by jsoref** [reply]
I'm fairly confident the answer is no. (I'd love to be proven wrong.)

Note that organizations are free (I probably have a dozen). You really should create an org for your action. It allows for better succession planning, better security, distinct github runner minutes, an overview of github runners.

- https://github.com/community/maintainers/discussions/662
